src/App.tsx

```diff
@@ -82,6 +84,9 @@ const App: React.FC = () => {
                       <Route path={Routes.VOLUNTEER}>
                         <Redirect to={Routes.LOGIN} />
                       </Route>
+                      <Route path={Routes.TEAM}>
+                        <Redirect to={Routes.LOGIN} />
+                      </Route>
```

Comment:
📌(out of scope for this PR, something to be done on scaffold) How can we create a catch-all which simply redirects the user to log in if they try to access a page other than Landing, Login, or Signup?

Reply:
Good point, since everything else redirects to Login, we could just only list the Landing, Login, and Signup routes (probably would want the Not Found one too) and then have a `<Redirect to={Routes.Login} />` after those four cases.

Reply:
This was a pretty quick change so I tested and added it while fixing the team ID parameter - can undo this easily too

Reply:
Agreed, that's a tradeoff for sure...  let's discuss next meeting